### DISCUSSION TITLE
Followed all instructions. Code dooesnt seem to run after auth file filled in

### DISCUSSION THREAD

**Discussion by sparten576** [DIGITALCRIMINALS/OnlyFans#1042] [Q&A]
I have filled in the auth file and trying running the OF only part of this. I have active subscriptions but it seems that it cant find anything. 
When looking for the cookie value, mine doesn't look like the example picture. Mine is missing some of the values in cookies.
![image](https://github.com/DIGITALCRIMINALS/UltimaScraper/assets/44298966/447adcac-e967-499d-90bc-0a8540b2be79)
Anyone help?

**Comment by Okekul1nko** [comment]
go to **\__user_data__\profiles\OnlyFans\default** and in **auth.json** change `"active": null` to `"active": true`
